### ISSUE TITLE
synchronize git

### ISSUE THREAD

**Issue by ktsaou** [benweet/stackedit#682]
Thank you for this amazing editor and viewer.
Excellent work. Thank you very much!

The only thing I miss, is git synchronization. For example, edit and save files directly on a github repo.

**Comment by sparkrod** [comment]
Agreed this would be a great feature. Also the ability to publish to private (non-github) git repos would be nice.

**Comment by johnjoepo** [comment]
Please add this Function!

**Comment by igregson** [comment]
+1  -- This would be sweet.

**Comment by jag3773** [comment]
:+1:

**Comment by briefpower** [comment]
+1   Please add this Function!

**Comment by evitolins** [comment]
+1 

or at least add Gitlab (and maybe Bitbucket too)

**Comment by valentijnvenus** [comment]
+1

**Comment by kahidna** [comment]
@benweet cool its working well, is there a way to integrate this feature to gitlab? or maybe good trick to sync between github and gitlab markdown?

**Comment by flxwu** [comment]
There is no gitlab integration in stackedit yet, but it could be added 😊
